### ISSUE TITLE
Training throws key error 

### ISSUE THREAD

**Issue by oneraghavan** [Hironsan/anago#49]
When Label present in validation set and not present in training set , the train method throws KeyError

**Comment by Hironsan** [comment]
Thanks @oneraghavan .
I will fix the problem.
